src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Samples.cshtml.cs

```diff
@@ -0,0 +1,211 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using ApiView;
+using APIView;
+using APIViewWeb.Models;
+using APIViewWeb.Repositories;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.Azure.Cosmos;
+using Microsoft.Extensions.Configuration;
+
+namespace APIViewWeb.Pages.Assemblies
+{
+    public class UsageSamplePageModel : PageModel
+    {
+        private readonly UsageSampleManager _samplesManager;
+        private readonly ReviewManager _reviewManager;
+        private readonly CommentsManager _commentsManager;
+        private readonly NotificationManager _notificationManager;
+        public readonly UserPreferenceCache _preferenceCache;
+        private readonly IAuthorizationService _authorizationService;
+
+        public string Endpoint { get; }
```

Comment:
`Endpoint` is unused property. right?

Reply:
Oh, yes, removed and pushed now, along with a few unused properties I saw while there